pyproject.toml

```diff
@@ -1,9 +1,10 @@
 [tool.poetry]
 name = "jgt_tools"
-version = "0.1.1"
+version = "0.1.2"
 description = "A collection of tools for commmon package scripts"
 authors = ["Brad Brown <brad.brown@rackspace.com>"]
 license = "MIT"
+readme = "README.rst"
```

Comment:
Just to double check, we don't need any pyproject.toml changes to pick up the data defaults because it's auto scanned by being in it's new location (I am having setup.py flash backs I think)

Reply:
Correct, as it is part of the auto-included package-name subdir, we get it for "free" (checked this by looking inside the tar generated from `poetry build` as well as testing an install of that tar in an in-progress PR for `tableread` to integrate this package).